src/js/components/modals/AppModalComponent.jsx

```diff
@@ -61,28 +61,19 @@ var AppModalComponent = React.createClass({
     Mousetrap.bind(["command+enter", "ctrl+enter"], () => this.handleSubmit());
 
     if (!this.hasInfoObject()) {
-      InfoStore.on(InfoEvents.CHANGE, this.onInfoChange);
-      InfoStore.on(InfoEvents.REQUEST_ERROR, this.onInfoChange);
+      InfoStore.once(InfoEvents.CHANGE, this.onInfoChange);
```

Comment:
Yes, sure, I mean if the component is unmounted, how do you ensure that the listeners isn't bound anymore?